src/admin/gutenberg/sesamy-post-editor/src/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const defaultConfig = require('@wordpress/scripts/config/webpack.config');
```

Comment:
I'm not sure if this makes a difference? We didn't have a webpack config file before..